pkgs/development/libraries/aspell/dictionaries.nix

```diff
@@ -41,10 +41,12 @@ let
     stdenv.mkDerivation ({
       name = "aspell-dict-${shortName}";
 
-      buildInputs = [aspell which];
+      nativeBuildInputs = [ aspell which ];
```

Comment:
when doing this kind of changes, adding `strictDeps = true;` helps avoid unintentional regression